ui/app/AppLayouts/WalletV2/components/AddAccountPopup.qml

```diff
@@ -0,0 +1,227 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Dialogs 1.3
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+
+ModalPopup {
```

Comment:
This should be a `StatusModal` from `StatusQ`

Reply:
done